src/main/java/de/dhbw/karlsruhe/domain/services/PlayDialogService.java

```diff
@@ -0,0 +1,89 @@
+package de.dhbw.karlsruhe.domain.services;
+
+import de.dhbw.karlsruhe.domain.models.Difficulty;
+import de.dhbw.karlsruhe.domain.models.Sudoku;
+import de.dhbw.karlsruhe.domain.models.generation.SudokuGeneratorBacktracking;
+import de.dhbw.karlsruhe.domain.models.generation.SudokuGeneratorTransformation;
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.Random;
+
+public class PlayDialogService {
+    Sudoku sudoku;
+    SudokuGeneratorTransformation sgTransformation = new SudokuGeneratorTransformation();
+    SudokuGeneratorBacktracking sgBacktracking = new SudokuGeneratorBacktracking();
+    SudokuValidatorService sudokuValidator = new SudokuValidatorService();
+    Random rand = new Random();
+
+    public PlayDialogService(){
+    }
+
+    public void startNewGame(Difficulty dif) {
+        if (rand.nextInt()<0.5){
+            System.out.println("Transformed sudoku generated:");
+            sudoku = sgTransformation.generateSudoku(dif);
+        } else {
+            System.out.println("Backtracking sudoku generated:");
+            sudoku = sgBacktracking.generateSudoku(dif);
+        }
+        System.out.println("Enter numbers by writing: W:[Row],[Column],[Value]");
+        System.out.println("Example: W:3,4,9");
+        System.out.println("To remove a number write: R:[Row],[Column]");
+        System.out.println("Example: R:3,4");
+        System.out.println("Initially filled fields can't be removed.");
+        while (sudokuValidator.isSudokuFinished(sudoku.getGameField())) {
+            sudoku.print();
+            userInputDialog();
+        }
+    }
+
+    private void userInputDialog() {
+        String input = ScannerService.getScanner().nextLine();
+        while (!inputCorrect(input)) {
+            System.out.println("The input did not match the input format.");
+            System.out.println("Enter numbers by writing: W:[Row],[Column],[Value]");
+            System.out.println("To remove a number write: R:[Row],[Column]");
+            input = ScannerService.getScanner().nextLine();
+        }
+        String[] getAction = input.split(":");
+
+        int[] splitInput = Arrays.stream(getAction[1].split(",")).mapToInt(Integer::parseInt).toArray();
```

Comment:
Möglicher IndexOutOfBounce? Käme zwar vorher schon in der inputCorrect Methode...